examples/README.md

```diff
@@ -0,0 +1,53 @@
+gRPC in 3 minutes (Go)
+======================
+
+BACKGROUND
+-------------
+For this sample, we've already generated the server and client stubs from [helloworld.proto](https://github.com/grpc/grpc-common/blob/master/protos/helloworld.proto). 
+
+PREREQUISITES
+-------------
+
+- This requires Go 1.4
```

Comment:
I was just copying the files from the `grpc-common` repo. Didn't make any change.